examples/webhook-signing/deno/main.ts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env -S npm run-script run
+
+import {serveListener} from 'https://deno.land/std@0.116.0/http/server.ts';
+import 'https://deno.land/x/dotenv/load.ts';
+
+import Stripe from 'npm:stripe@^11.16';
+
+const stripe = Stripe(Deno.env.get('STRIPE_API_KEY'));
+
+// Must specify hostname explicitly, see https://github.com/denoland/deno/issues/5144
+const server = Deno.listen({hostname: '127.0.0.1', port: 8080});
```

Comment:
can we bind to port 0 (dynamic) and retrieve the actual value from the server?

Reply:
Yes! Added in latest commit